doc/green_modes/green_modes_server.rst

```diff
@@ -26,11 +22,10 @@ as it is a lot harder to debug. You should use this green mode with care.
 change too much in your existing code (or you don't feel comfortable with
 writing syntax of asynchronous calls).
 
-Another thing to have in mind is that the Tango monitor lock is present - you
-can't have two read operations happening concurrently. Any subsequent ones
-will always have to wait for the first one to finish.
+Another thing to keep in mind is that when using :obj:`~tango.GreenMode.Gevent`
+green mode there is no Tango monitor lock!
 
-Greenlets (a task in a background, but handled within the event loop) can be
+Greenlets (a task in the background, but handled within the event loop) can be
```

Comment:
Two suggestions:

     Another thing to keep in mind is that when using :obj:`~tango.GreenMode.Gevent` green mode 
     the Tango monitor lock is disabled, so the client requests can be processed concurrently.

and: 

    Greenlets can also be used to spawn tasks in the background.

Reply:
Thanks.  Updated in 9e0a2d9.